packages/instant/package.json

```diff
@@ -23,6 +19,7 @@
         "clean": "shx rm -rf lib coverage scripts",
         "deploy_dogfood": "discharge deploy -c .dogfood.discharge.json",
         "deploy_staging": "discharge deploy -c .staging.discharge.json",
+        "deploy_live": "discharge deploy -c .production.discharge.json",
```

Comment:
nit but I would have expected this to be called `deploy_production`

Reply:
yea... it's meant to match the website but on second thought that isn't a good enough reason.